CMakeLists.txt

```diff
@@ -115,7 +115,7 @@ dependent_option(ENABLE_INTERNAL_FLATBUFFERS "Enable internal flatbuffers?")
 dependent_option(ENABLE_INTERNAL_FMT "Enable internal fmt?")
 dependent_option(ENABLE_INTERNAL_NFS "Enable internal libnfs?")
 dependent_option(ENABLE_INTERNAL_PCRE2 "Enable internal pcre2?")
-dependent_option(ENABLE_INTERNAL_RapidJSON "Enable internal rapidjson?")
+dependent_option(ENABLE_INTERNAL_NlohmannJSON "Enable internal nlohmannjson?")
```

Comment:
I would make this all uppercase. It makes things easier internally to rely on that standard, and was changed at some point in time prior to my involvement to be the defacto